app/src/main/java/se/inera/intyg/minaintyg/util/html/HTMLTextFactory.java

```diff
@@ -0,0 +1,33 @@
+package se.inera.intyg.minaintyg.util.html;
+
+import static se.inera.intyg.minaintyg.util.html.HTMLFactory.tag;
+
+public class HTMLTextFactory {
```

Comment:
Add private constructors and throw new IllegalStateException("Utility class"); if used.